getting_started/create_vm/make_vm.sh

```diff
@@ -2,19 +2,25 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the Apache 2.0 License.
 
-set -ex
+set -e
+
+if [ "$#" -lt 1 ]; then
+  echo "make_vm.sh takes at least an Azure subscription name,"
+  echo "and an optional path to an SSH public key"
+  exit 1
+fi
 
 # Azure Resource Group
-RG=ccf-dev
+RG=ccf-dev-rg
 # Region, only eastus and westeurope are supported at the moment
-REGION=eastus
+REGION=uksouth
 CFG=vm.json
 
 # By default, use current ssh public key
 DEFAULT_PUBLIC_KEY_PATH=~/.ssh/id_rsa.pub
-PUBLIC_KEY_PATH=${1:-${DEFAULT_PUBLIC_KEY_PATH}}
+PUBLIC_KEY_PATH=${2:-${DEFAULT_PUBLIC_KEY_PATH}}
 
 az login
-az group create --name "$RG" --location "$REGION" --subscription "$SUBSCRIPTION"
+az group create --name "$RG" --location "$REGION" --subscription "$1"
 oe-engine generate --api-model "$CFG" --ssh-public-key "$PUBLIC_KEY_PATH" --output-directory generated
-az group deployment create --name "$RG-$REGION" --resource-group "$RG" --template-file generated/azuredeploy.json --parameters @generated/azuredeploy.parameters.json --subscription "$SUBSCRIPTION"
+az deployment group create --name "$RG-$REGION" --resource-group "$RG" --template-file generated/azuredeploy.json --parameters @generated/azuredeploy.parameters.json --subscription "$1"
```

Comment:
`group deployment` is deprecated and has been replaced by `deployment group` (yes, really).